storage/gcloud/aio/storage/storage.py

```diff
@@ -100,8 +100,10 @@ def get_bucket(self, bucket_name: str) -> Bucket:
         data: dict = await resp.json()
         return data
 
+    # pylint: disable-msg=too-many-locals
```

Comment:
We had 15 local variables in total with the limit of 15, after I've added "parameters" pylint started to complain. Decided to suppress warning for now as changing the interface or splitting the method can be a separate refactoring commit, but can include everything to this pull request if we want to

Reply:
Sounds good to me. nit: please use `disable` instead of `disable-msg` -- the latter has been deprecated as of pylint 0.21.0.

Reply:
changed in the new commit